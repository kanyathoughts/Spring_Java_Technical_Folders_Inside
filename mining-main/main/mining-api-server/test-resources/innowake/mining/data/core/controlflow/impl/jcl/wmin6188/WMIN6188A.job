//MATEKSD JOB MSGLEVEL=(1,1),NOTIFY=&SYSUID
//*
//* EXAMPLE TO SHOW IF CONDITION IN JCL
//*
//STEP01   EXEC PGM=IFCOND1
//*
//IFSTMT1 IF STEP01.RC = 0  THEN  
//STEP02   EXEC PGM=IFCOND2             
//STEP03   EXEC PGM=IFCOND3             
//        ENDIF
//STEP04   EXEC PGM=IFCOND4
//STEP05   EXEC PGM=IFCOND5
//*
//IFSTMT2 IF STEP05.RC = 04 THEN   
//STEP06   EXEC PGM=IFCOND6             
//        ELSE                          
//STEP07   EXEC PGM=IFCOND7
//        ENDIF
//STEP08   EXEC PGM=IFCOND8
//*