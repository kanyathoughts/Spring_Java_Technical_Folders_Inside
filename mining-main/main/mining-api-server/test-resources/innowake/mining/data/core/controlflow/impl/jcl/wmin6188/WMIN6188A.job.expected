 #2 JclJobNode, Label: MATEKSD {NOTIFY=&SYSUID, MSGLEVEL=1,1}, FROM={TestEntryPoint}, TO={#3}
  #3 JclStepExec, Label: STEP01 EXEC PGM=IFCOND1, FROM={#2}, TO={#4}
  #4 JclStepIf, Label: IFSTMT1 IF STEP01.RC EQ 0, FROM={#3}, TO={#7(TRUE), #9(FALSE)}
   #5 JclCondition, Label: STEP01.RC EQ 0
    #7 JclStepExec, Label: STEP02 EXEC PGM=IFCOND2, FROM={#4(TRUE)}, TO={#8}
    #8 JclStepExec, Label: STEP03 EXEC PGM=IFCOND3, FROM={#7}, TO={#9}
  #9 JclStepExec, Label: STEP04 EXEC PGM=IFCOND4, FROM={#4(FALSE), #8}, TO={#10}
  #10 JclStepExec, Label: STEP05 EXEC PGM=IFCOND5, FROM={#9}, TO={#11}
  #11 JclStepIf, Label: IFSTMT2 IF STEP05.RC EQ 4, FROM={#10}, TO={#14(TRUE), #16(FALSE)}
   #12 JclCondition, Label: STEP05.RC EQ 4
    #14 JclStepExec, Label: STEP06 EXEC PGM=IFCOND6, FROM={#11(TRUE)}, TO={#17}
    #16 JclStepExec, Label: STEP07 EXEC PGM=IFCOND7, FROM={#11(FALSE)}, TO={#17}
  #17 JclStepExec, Label: STEP08 EXEC PGM=IFCOND8, FROM={#14, #16}, TO={TestReturnPoint}
